example/config/experiment.yaml

```diff
@@ -327,8 +327,14 @@ bias_audit_config:
 # Rank tiebreaking - In the predictions.rank_abs and rank_pct columns, ties in the score
 # are broken either at random or based on the 'worst' or 'best' options. 'worst' is the default.
 #
-# 'worst' will break ties with the ascending label value, so if you take the top 'k' predictions, and there are ties across the 'k' threshold, the predictions above the threshold will be negative labels if possible.
-# 'best' will break ties with the descending label value, so if you take the top 'k' predictions, and there are ties across the 'k' threshold, the predictions above the threshold will be positive labels if possible.
+# 'worst' will break ties with the ascending label value, so if you
+# take the top 'k' predictions, and there are ties across the 'k'
+# threshold, the predictions above the threshold will be negative
+# labels if possible. NULL labels are also pushed to the top.
+# 'best' will break ties with the descending label value, so if you
+# take the top 'k' predictions, and there are ties across the 'k'
+# threshold, the predictions above the threshold will be positive
+# labels if possible. NULL labels are pushed to the bottom.
```

Comment:
This makes more sense than what was happening before to be sure, but it's not entirely clear to me that nulls first reflects the "worst case" rather than, say, 0's then NULLs, then 1's? We'd have to write our own sorting functionality to do that and I'm not sure I feel strongly enough here to make that worthwhile...

Reply:
It is worth to explore. I see your point. I will open an issue, and tag it with `needs-discussion`

Reply:
Opened in #801